x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/_field_data_card.scss

```diff
@@ -1,14 +1,15 @@
 .mlFieldDataCard {
-  height: 420px;
-  width: 360px;
+  height: 480px;
+  box-shadow: none;
+  border-color: $euiColorLightShade;
```

Comment:
FYI, there is a global variable for border colors `$euiBorderColor`

Reply:
Thanks, forgot about this.